<launch>
    <node pkg="ssafy_2" type="mgeo_pub.py" name="mgeo_pub"  />
    <node pkg="ssafy_2" type="gpsimu_parser.py" name="gpsimu_parser" />
    <node pkg="ssafy_2" type="global_path_pub.py" name="global_path_pub"  />
    <node pkg="ssafy_2" type="local_path_pub.py" name="local_path_pub" />    
    <node pkg="ssafy_2" type="tf_pub.py" name="tf"  />
   
    <node pkg="ssafy_2" type="acc.py" name="advanced_purepursuit"  args="/lattice_path"/>
    
    <node pkg="ssafy_3" type="lattice_planner.py" name="lattice_planner" output="screen" args="/Object_topic"/>
    <node pkg="ssafy_3" type="lane_change_3.py" name="lane_change" args="/Object_topic"/>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find ssafy_3)/rviz/lane_detection_rviz.rviz" />
</launch>